packages/generate-persisted-query-manifest/src/index.ts

```diff
@@ -482,13 +500,20 @@ export async function generatePersistedQueryManifest(
       continue;
     }
 
+    let documentCount = 0;
+
     // We delegate validation to the functions that return the document sources.
     // We just need to record the operations here to sort them in the manifest
     // output.
     visit(source.node, {
       OperationDefinition(node) {
         const name = node.name?.value;
 
+        if (++documentCount > 1) {
+          addError(source, ERROR_MESSAGES.multipleOperations());
```

Comment:
Should we put the error on the second operation node as opposed to on "source" which appears in test to mean that all errors are showing up as `1:1`?

Reply:
Looking back through the implementation of [`addError`](https://github.com/apollographql/apollo-utils/blob/930dee9b49e0d143e80aa610c68474ff2fed453e/packages/generate-persisted-query-manifest/src/index.ts#L266-L272), it looks like we really don't care about `node` in that output and only use `file` and `location` as part of the error message, so it doesn't look like it would matter which `node` we use here.

On the note about errors 1:1 with operation nodes, I considered that (and actually implemented that first), but I found this only makes sense if we include the operation name in the error output, otherwise the error just looks repetitive. Here is what something like that looks like:

```ts
  expect(stderr).toMatchInlineSnapshot(`
    [
      "src/mixed.graphql",
      "1:1  error  Cannot declare multiple operations in a single document. Please remove the 'TestQuery' operation or move it into a different document.",
      "1:1  error  Cannot declare multiple operations in a single document. Please remove the 'TestSubscription' operation or move it into a different document.",
      "src/mutations.graphql",
      "1:1  error  Cannot declare multiple operations in a single document. Please remove the 'SayGoodbye' operation or move it into a different document.",
      "src/queries.graphql",
      "1:1  error  Cannot declare multiple operations in a single document. Please remove the 'GoodbyeQuery' operation or move it into a different document.",
      "src/subscriptions.graphql",
      "1:1  error  Cannot declare multiple operations in a single document. Please remove the 'GoodbyeSubscription' operation or move it into a different document.",
      "✖ 5 errors",
    ]
  `);

```

vs what it is now:

```ts
  expect(stderr).toMatchInlineSnapshot(`
    [
      "src/mixed.graphql",
      "1:1  error  Cannot declare multiple operations in a single document.",
      "src/mutations.graphql",
      "1:1  error  Cannot declare multiple operations in a single document.",
      "src/queries.graphql",
      "1:1  error  Cannot declare multiple operations in a single document.",
      "src/subscriptions.graphql",
      "1:1  error  Cannot declare multiple operations in a single document.",
      "✖ 4 errors",
    ]
  `);
```

Thoughts?